.github/workflows/docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+name: Test Docker image
+
+on:
+  push:
+    branches:
+      - 'main'
+    tags:
+      - '*'
+
+jobs:
+  docker-compose:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Launch Attack Flow using Docker Compose
```

Comment:
I'm not familiar with this action. Does this build the container? Does it also run `docker compose up`?

Reply:
@banzo Can you please answer this?

Reply:
I did add a comment butI am not sure yet why it does not appear here: https://github.com/cetic/attack-flow/blob/main/.github/workflows/docker-compose.yml#L12

Reply:
@banzo Thank you for the updates. 

I still don't understand the purpose of this github action. Is this to make sure the compose file is formatted correctly? Does it run forever or for a set amount of time?

Reply:
yes, it is a test that checks the docker-compose.yml file is working out of the box: it launches the stack (`docker-compose up`), checks that there are no errors, stops the stack, and finally does some cleanup.